lib/psulib_traject/holdings.rb

```diff
@@ -28,7 +34,9 @@ def initialize(record:, context:, classification:)
     def resolve_base
       return [] if online? || holdings.empty?
 
-      holdings.reject! { |call_number| call_number.exclude? || classification_not_requested?(call_number) }
+      holdings.reject! do |call_number|
+        call_number.exclude? || call_number.not_browsable? || classification_not_requested?(call_number)
```

Comment:
Sorry, this looks confusing now. I think `classification_not_requested?` can be deleted, as well as the `classification` parameter because we're not choosing which classifications we want indexed. Probably ditto for the `CLASSIFICATIONS` constant as well.

Reply:
ahh correct! I will remove them 👍🏻

Reply:
removed